### ISSUE TITLE
Tender location directly tied to tender

### ISSUE THREAD

**Issue by tdavis9** [open-contracting/standard#1103]
Makueni County, Kenya has tenders that are directly related to locations, regardless of the items within the tender. We are aware of the OCDS Location Extension but this assumes that items are linked to locations, which is not what we encountered. We have created an OCDS extension that depends and extends upon the OCDS Tender Location extension, but re-uses this entity directly at the tender level

https://github.com/devgateway/forms-makueni/tree/master/persistence-mongodb/src/main/resources/extensions/tender_location

**Comment by jpmckinney** [comment]
Thanks, @tdavis9. Can you share some sample location descriptors? Does Makueni County have item-level information?

If there is item-level information, the location data can be repeated across all items. To avoid this repetition, there is issue #893 about moving some reused content to the top-level. In that case, each item would just refer to the same location.

**Comment by jpmckinney** [comment]
@yolile @duncandewhurst This use case sounds familiar – not sure if it's come up in other implementations.

**Comment by jpmckinney** [comment]
@yolile I found one case: in #888, all items in a given lot had the same location. That case might also be addressed by #893 (without adding a location field to the lot).

**Comment by yolile** [comment]
@jpmckinney yes, that was the case in Paraguay. I recently have seen that in Peru the same happens with the contracts from a framework agreement, where the location is at contract level, example: https://apps1.perucompras.gob.pe//OrdenCompra/obtenerPdfOrdenPublico?ID_OrdenCompra=1109816&ImprimirCompleto=1

page 2

**Comment by mpostelnicu** [comment]
> Thanks, @tdavis9. Can you share some sample location descriptors? Does Makueni County have item-level information?
> 
> If there is item-level information, the location data can be repeated across all items. To avoid this repetition, there is issue #893 about moving some reused content to the top-level. In that case, each item would just refer to the same location.

@jpmckinney yes we do have items but since in some cases there can be a lot of them, we wanted to avoid repetition to keep the OCDS json smaller. #893 sounds like a great idea.

**Comment by duncandewhurst** [comment]
I found several examples of systems with location at the tender level.

* In NSW Australia's eTendering system, a tender's location can be a list of NSW regions and/or Australian states and territories ([example](https://www.tenders.nsw.gov.au/dcs/?event=public.rft.show&RFTUUID=1EAEDEBA-A628-753D-0BD40E335A5129B8)).
* In Australia's AusTender system, a tender's location can be a list of Australian states and territories and/or cities ([example](https://www.tenders.gov.au/Atm/Show/6cecee54-48b0-4869-ab36-8198907c7ae6)).
* In New Zealand's GETS system, a tender's location can be a list of New Zealand regions ([example](https://www.gets.govt.nz/NRC/ExternalTenderDetails.htm?id=25075262))
* In the UK's Contracts Finder system, a tender's location can be a lat/long, a postcode or a region ([mapping](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/936537/Input-Output_Field_Mapping_Contracts_Finder_OCDS__3_.ods))

Similar to the relationship between the tender classification extension and the item classifications, I think this points to a need for a tender location in addition to item locations.

**Comment by jpmckinney** [comment]
What is the meaning of those locations? Delivery locations like in the location extension?

On that understanding, I'll move this issue to ocds-extensions, as it relates to the Location extension.
